app/templates/route-components/progress-indicators.hbs

```diff
@@ -0,0 +1,71 @@
+<h1>Progress indicators</h1>
+<p>
+	Inform the user that an operation is in progress, to reassure that the system is not hung or waiting for user input,
+	and often to provide the user with an estimate of how far through a task the system has progressed.
+</p>
+
+<section>
+	<h2 id="loading-spinner">Loading spinner</h2>
+	<p>
+		A loading spinner is used best for short processes or when total time of completion is unknown.
+	</p>
+	<p>
+		The <code>wds-spinner</code> class should be used in conjunction with <code>wds-spinner__stroke</code>
+		and applied to an SVG with a circle. The properties of the SVG have to be either calculated beforehand
+		or via JavaScript code.
+	</p>
+	{{#component-demo}}
+		<!--size = full diameter = r * 2 + stroke width-->
+		<svg class='wds-spinner wds-spinner__block' width='78' height='78'
+			 viewBox='0 0 78 78' xmlns='http://www.w3.org/2000/svg'>
+			<!--transform over full radius = r  + strokeWidth / 2-->
+			<g transform='translate(39, 39)'>
+				<!--stroke-dasharray and stroke-dashoffset="238" are equal to 2*PI*r = 2*PI*38-->
+				<circle class='wds-spinner__stroke' fill='none' stroke-width='2'
+						stroke-dasharray="239" stroke-dashoffset="239"
+						stroke-linecap='round' r='38'></circle>
+			</g>
+		</svg>
+	{{/component-demo}}
+	<div>
+		In addition to <code>wds-spinner</code> and <code>wds-spinner__stroke</code>, other classes can be used,
+		to modify spinner's appearance:
+		<ul>
+			<li>
+				<code>wds-spinner__block</code>, will make the spinner be displayed as a block
+				- useful if you intent to use the spinner as a list element,
+			</li>
+			<li>
+				<code>wds-spinner__overlay</code>, will cause the spinner to display a translucent
+				white background and position it fixed in the center,
+			</li>
+			<li>
+				<code>wds-spinner__stroke-theme-color</code>, which should be defined in your application
```

Comment:
In Mercury some of the spinners have been themed and some have been not. Since this class does not exist in Design System I can remove this piece of documentation and code related to that.